github-users-web-client/src/main/java/com/xemantic/githubusers/web/navigation/UserSelectionNavigator.java

```diff
@@ -37,22 +37,21 @@
 @Singleton
 public class UserSelectionNavigator {
 
-  private final EventBus eventBus;
+  private final Observable<UserSelectedEvent> eventBus;
```

Comment:
Yes, you really should change it. I just leave it as eventBus to remark that this is almost the same, just a bit specific type. But, yep, after that dummy hint 😉, the variable should be renamed.

My conclusions about $ is that... it is especially useful in local variables because if you do this
```
void someMethod() {
  Observable<String> tick$ = ...;
  tick$.subscribe(tick -> ...);
}
```
And you call the observable `tick` you cannot use `tick` as lambda argument which is sometimes a bit annoying, also the $ can be read as the plural of `tick` which "sounds" good. BUT, also abusing of the $ might be annoying too. So... hehe be careful, I think I'll use userSelectedBus` as simple conventions without the risk of abusing on $, so `{event name without the Event end}Bus`. But this is your project 😜.